timesys/high-level-triggers.py

```diff
@@ -1041,6 +1041,140 @@
         },
         'channels': ('IA-01RaInj05:PU-Osc:AUXIN', ),
         },
+    'AS-Glob:TI-RaBPM-Dbg': {
+        'database': {
+            'Src': {'enums': ('Linac', 'Cycle', 'Study', 'OrbSI', 'FOFBS', 'Debug', )},
```

Comment:
You don't need to add `'Linac'` and `'Study'` here.

The events `'RmpBO', 'Linac', 'Study'` are added to all triggers.

Why do you need `'Cycle'` here?

Reply:
> Why do you need 'Cycle' here?

Kind of for completeness, but also as a bit of (possibly unnecessary) future proofing, I think. It would be for some possible time when the fast power supplies are cycled using triggers instead of via EPICS PVs only.